.travis.yml

```diff
@@ -11,8 +11,33 @@ env:
   - KARTOTHEK_ARROW_VERSION=0.12.1
   - KARTOTHEK_ARROW_VERSION=0.13.0
 
+
+matrix:
+  allow_failures:
+    - name: "Integration test for Numpy Pandas"
+    - name: "Integration test for PyArrow"
+  include:
+    - name: "Integration test for Numpy Pandas"
+      if: branch=master
+      python: "3.7"
+      env:
+        - NUMPY_PANDAS_INTEGRATION=1
```

Comment:
Where is this environment variable being used?

Reply:
@kagharpure, Initially we have used that variable for enabling some conditions but now we can remove that.